vector/src/test/java/im/vector/app/features/location/live/map/UserLiveLocationViewStateMapperTest.kt

```diff
@@ -79,14 +84,16 @@ class UserLiveLocationViewStateMapperTest {
         val viewState = userLiveLocationViewStateMapper.map(summary)
 
         val expectedViewState = UserLiveLocationViewState(
-                userId = A_USER_ID,
+                matrixItem = MatrixItem.UserItem(id = A_USER_ID, displayName = "User 2", avatarUrl = ""),
                 pinDrawable = pinDrawable,
                 locationData = LocationData(
                         latitude = A_LATITUDE,
                         longitude = A_LONGITUDE,
                         uncertainty = A_UNCERTAINTY
                 ),
-                endOfLiveTimestampMillis = A_END_OF_LIVE_TIMESTAMP
+                endOfLiveTimestampMillis = A_END_OF_LIVE_TIMESTAMP,
+                locationTimestampMillis = A_END_OF_LIVE_TIMESTAMP,
```

Comment:
I think here we should declare a dedicated location timestamp that should be set in the given `locationDataContent` variable. So that the mapping is checked.

Reply:
Done.